### PR TITLE
101 webapp linux django - new template azure4students

### PR DESCRIPTION
**Pull request by jose-mart**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by jose-mart** [comment]
Ci-USGov is failing because it is trying to deploy the template to an invalid location.
image
Microsoft Web Sites don't look available for US gov.

File: 101-webapp-linux-django/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-webapp-linux-django%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a>
+
+#### Important disclaimer: Azure charges you for the resources you are using, and you don't want to finish all your credits at once, right? So, don't forget to stop the ___Service___ at the portal or even delete the Resource Group you have created to avoid unnecessary charges
+
+### Deploy your code
+
+It will be really easy. First of all, download the basic Django project configuration from [Azure-Sample](https://github.com/Azure-Samples/docker-django-webapp-linux.git). You can clone the repository using git.
```
Comment (qubitron) [inline 101-webapp-linux-django/GettingStarted.md]: This sample doesn't use a production ready Dockerfile, you can reference our VS Code django + docker sample here:
https://code.visualstudio.com/docs/python/tutorial-create-containers#_changes-for-django-apps

Official sample app here:
https://github.com/Microsoft/python-sample-vscode-django-tutorial

Comment (jose-mart) [reply 101-webapp-linux-django/GettingStarted.md]: Thank you for the feedback. I updated the sample app with the official one.

File: 101-webapp-linux-django/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
+{
+    "contentVersion": "1.0.0.0",
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
+            "defaultValue": "[concat('Django-', uniqueString(resourceGroup().name, utcNow('F')))]",
```
Comment (bmoore-msft) [inline 101-webapp-linux-django/azuredeploy.json]: make this idempotent - use resourceGroup().id and remove utcnow()

File: 101-webapp-linux-django/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    "variables": {
+        "sku": "Free",
+        "skuCode": "F1",
+        "workerSize": "0",
+        "appInsights": "[concat(parameters('webAppName'), '-insights')]",
+        "hostingPlanName": "[concat( 'hpn-', resourceGroup().name)]"
+    },
+    "resources": [
+        {
+            "name": "[parameters('webAppName')]",
```
Comment (bmoore-msft) [inline 101-webapp-linux-django/azuredeploy.json]: see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

File: 101-webapp-linux-django/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            "properties": {
+                "siteConfig": {
+                    "appSettings": [
+                        {
+                            "name": "WEBSITES_ENABLE_APP_SERVICE_STORAGE",
+                            "value": "false"
+                        },
+                        {
+                            "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                            "value": "[reference(resourceId('microsoft.insights/components/', variables('appInsights')), '2018-05-01-preview').InstrumentationKey]"
```
Comment (bmoore-msft) [inline 101-webapp-linux-django/azuredeploy.json]: remove the apiVersion

File: 101-webapp-linux-django/azuredeploy.json (hunk 2 above)
Comment (jose-mart) [reply 101-webapp-linux-django/azuredeploy.json]: Updated

File: 101-webapp-linux-django/azuredeploy.json (hunk 3 above)
Comment (jose-mart) [reply 101-webapp-linux-django/azuredeploy.json]: Sorted properties

File: 101-webapp-linux-django/azuredeploy.json (hunk 4 above)
Comment (jose-mart) [reply 101-webapp-linux-django/azuredeploy.json]: Removed

File: 101-webapp-linux-django/metadata.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "itemDisplayName": "Django app",
+    "description": "This template allows you to create an App Service for deploying a Django app.", 
+    "summary": "Deploy an App Service that supports Python. It gives you the support to deploy your own Django app.",
+    "githubUsername": "leestott",
+    "tags":{
+      "subscriptiontype": "Azure4Student",
+      "resource":"appServices, python, django",
+      "os": "linux",
+      "level": "beginner",
+      "cost": "$"
```
Comment (bmoore-msft) [inline 101-webapp-linux-django/metadata.json]: the other free skus have this set as "$0"

**Pull request merged**